sktime/registry/_base_classes.py

```diff
@@ -261,11 +261,11 @@ def get_base_class(cls):
         return BaseRegressor
 
 
-class series_annotator(_BaseScitypeOfObject):
+class detector(_BaseScitypeOfObject):  # todo 0.37.0: remove series-annotator
```

Comment:
both tags need to be present, although I would list only the `detector` in the API reference. Please also have a look at the tags api reference, in `tags.rst`.

Reply:
@fkiraly thanks for review, so shall I change ` """Time series annotator."""` to ` """Time series detectorr."""` in line `265`? can you please suggest other tags necessary?